idea/src/META-INF/plugin.xml

```diff
@@ -1554,6 +1554,11 @@
       <category>Kotlin</category>
     </intentionAction>
 
+    <intentionAction>
```

Comment:
Really inspection is enough here. Since IDEA 2016.3, we have `INFORMATION` level for inspection (= No highlighting, only fix), so separate intentions are not necessary in this situation.